python/py-nbxmpp/Portfile

```diff
@@ -1,35 +1,39 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
-PortSystem          1.0
-PortGroup           python 1.0
+PortSystem           1.0
```

Comment:
do not change whitespacing, especially not since it now doesn't conform to the modeline anymore. If needed, formatting changes should be done separately from functional changes.

Reply:
Reverted back.